Metrics/setMinkowski.m

```diff
@@ -4,35 +4,32 @@
 %   INPUTS:
 %   gridSize - World size in [t,x,y,z]
 %
-%   metric - Metric struct
-%
 %   OUTPUTS:
-%   tensor - The metric tensor as a 4x4 cell of 4D arrays.
+%   metric - Metric struct containing the metric tensor as a 4x4 cell of 4D arrays.
 
-%%
+% Check input gridSize
+if ~isvector(gridSize) || length(gridSize) ~= 4
+    error("gridSize must be a 4-element vector [t, x, y, z].");
+end
 
-% dt^2 term
-metric{1,1} = -ones(gridSize);
+% Initialize metric struct
```

Comment:
The intent for this function is not to return a struct. Use Metrics/Minkowski/metricGet_Minkowski.m instead.